test/dotnet-new3.UnitTests/DotnetNewUpdateCheck.cs

```diff
@@ -111,6 +111,36 @@ public void PrintInfoOnUpdateOnCreation()
                   .And.HaveStdOutMatching("   dotnet new3 --install Microsoft.DotNet.Common.ProjectTemplates.5.0::([\\d\\.a-z-])+");
         }
 
+        [Fact]
+        public void DoesNotPrintUpdateInfoOnCreation_WhenNoUpdateCheckOption()
+        {
+            var home = TestUtils.CreateTemporaryFolder("Home");
+            new DotnetNewCommand(_log, "-i", "Microsoft.DotNet.Common.ProjectTemplates.5.0::5.0.0")
+                .WithCustomHive(home).WithoutBuiltInTemplates().Quietly()
+                .WithWorkingDirectory(TestUtils.CreateTemporaryFolder())
+                .Execute()
+                .Should()
+                .ExitWith(0)
+                .And
+                .NotHaveStdErr()
+                .And.HaveStdOutContaining("Success:")
+                .And.HaveStdOutContaining("console")
+                .And.HaveStdOutContaining("classlib");
+
+            new DotnetNewCommand(_log, "console", "--no-update-check")
```

Comment:
Can you just add 1 more execution without `--no-update-check` and Assert that `"An update for template package 'Microsoft.DotNet.Common.ProjectTemplates.5.0::5.0.0' is available"` text is present?

Reply:
Preferably without specific version in package name, so just `"An update for template package 'Microsoft.DotNet.Common.ProjectTemplates.*is available"`